designer/server/src/createServer.ts

```diff
@@ -84,6 +84,12 @@ export async function createServer() {
   server.method('session.set', (id, value) => cache.set(id, value))
   server.method('session.drop', (id) => cache.drop(id))
 
+  server.method('state.get', (userId, key) => cache.get(`${userId}.${key}`))
+  server.method('state.set', (userId, key, value, ttl = config.sessionTtl) =>
```

Comment:
Do we definitely need to set the default if it's already the default?

```suggestion
  server.method('state.set', (userId, key, value, ttl) =>
```

Reply:
I missed this before. true, I agree. I will remove default.

Reply:
done